tests/routers/test_statistics.py

```diff
@@ -0,0 +1,434 @@
+
+    assert response.status_code == 200
+    assert TEST_MINIMAL_SEQ_SCORESET["targetGenes"][0]["targetSequence"]["reference"]["organismName"] in response.json()
+    assert (
+        response.json()[TEST_MINIMAL_SEQ_SCORESET["targetGenes"][0]["targetSequence"]["reference"]["organismName"]] == 1
+    )
+
+
+def test_target_gene_reference_statistics_acc(client, setup_router_db, data_files):
+    """Test target gene reference statistics on accession based targets for published score sets."""
+    experiment = create_experiment(client)
+    with patch.object(cdot.hgvs.dataproviders.RESTDataProvider, "_get_transcript", return_value=TEST_CDOT_TRANSCRIPT):
+        score_set = create_acc_score_set_with_variants(client, experiment["urn"], data_files / "scores_acc.csv")
+
+    publish_score_set(client, score_set["urn"])
+    response = client.get("/api/v1/statistics/target/gene/reference")
+
+    assert response.status_code == 200
+    assert TEST_MINIMAL_ACC_SCORESET["targetGenes"][0]["targetAccession"]["assembly"] in response.json()
+    assert response.json()[TEST_MINIMAL_ACC_SCORESET["targetGenes"][0]["targetAccession"]["assembly"]] == 1
+
+
+def test_target_gene_reference_statistics_seq(client, setup_router_db, data_files):
+    """Test target gene reference statistics on sequence based targets for published score sets."""
+    experiment = create_experiment(client)
+    score_set = create_seq_score_set_with_variants(client, experiment["urn"], data_files / "scores.csv")
+
+    publish_score_set(client, score_set["urn"])
+    response = client.get("/api/v1/statistics/target/gene/reference")
+
+    assert response.status_code == 200
+    assert TEST_MINIMAL_SEQ_SCORESET["targetGenes"][0]["targetSequence"]["reference"]["shortName"] in response.json()
+    assert response.json()[TEST_MINIMAL_SEQ_SCORESET["targetGenes"][0]["targetSequence"]["reference"]["shortName"]] == 1
+
+
+# Target gene identifier based statistics behave similarly enough to parametrize.
+@pytest.mark.parametrize("field_value", TARGET_GENE_IDENTIFIER_FIELDS)
+def test_target_gene_identifier_statistiscs(client, setup_router_db, field_value, data_files):
+    """Test target gene identifier statistics endpoint for published score sets."""
+    # Helper map that simplifies parametrization.
+    test_mapper = {
+        "ensembl-identifier": {"offset": 0, "identifier": {"dbName": "Ensembl", "identifier": "ENSG00000103275"}},
+        "refseq-identifier": {"offset": 0, "identifier": {"dbName": "RefSeq", "identifier": "NM_003345"}},
+        "uniprot-identifier": {"offset": 0, "identifier": {"dbName": "UniProt", "identifier": "Q9Y617"}},
+    }
+
+    experiment = create_experiment(client)
+
+    # Update each of the score set target genes to include our test external identifiers.
+    seq_target = TEST_MINIMAL_SEQ_SCORESET["targetGenes"][0].copy()
+    seq_target.update({"externalIdentifiers": [v for v in test_mapper.values()]})
```

Comment:
can prolly extract `{"externalIdentifiers": [v for v in test_mapper.values()]}` into a local var